File: stud_aid/lib/models/reservation.dart
import 'package:json_annotation/json_annotation.dart';

part 'reservation.g.dart';

@JsonSerializable()
class Reservation {
  int? reservationId;
  int? advertId;
  int? userId;
  String? selectedTime;

  Reservation() {}
  factory Reservation.fromJson(Map<String, dynamic> json) =>
      _$ReservationFromJson(json);

  Map<String, dynamic> toJson() => _$ReservationToJson(this);
}

File: StudAid/StudAid.Model/Reservation.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace StudAid.Model
{
    public partial class Reservation
    {
        
        public int? AdvertId { get; set; }
  
        public int? UserId { get; set; }

        public string SelectedTime { get; set; }
    }
}
